### PR TITLE
YDMU 7 more

### PR DESCRIPTION
**Pull request by Simisays**
Still in draft because Slimefoot's spellbook hasn't been revealed. Will most likely be tomorrow

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/ghitu_embercoiler.txt [hunk 1]
```diff
@@ -0,0 +1,13 @@
+Name:Ghitu Embercoiler
+ManaCost:1 R
+Types:Creature Human Wizard
+PT:2/2
+K:Prowess
+T:Mode$ Phase | Phase$ Main1 | PreCombatMain$ True | ValidPlayer$ You | TriggerZones$ Battlefield | Execute$ TrigSeek | TriggerDescription$ At the beginning of your precombat main phase, you may discard a card. If you do, seek a card with greater mana value and exile it. Until the end of your next turn, you may play the exiled card. 
+SVar:TrigSeek:AB$ ChangeZone | Cost$ Discard<1/Card> | Origin$ Library | Destination$ Exile | AtRandom$ True | NoShuffle$ True | Mandatory$ True | NoLooking$ True | NoReveal$ True | ChangeType$ Card.cmcGTX | ChangeNum$ 1 | RememberChanged$ True | SubAbility$ DBEffect
+SVar:X:Discarded$CardManaCost
+SVar:DBEffect:DB$ Effect | RememberObjects$ Remembered | StaticAbilities$ MayPlay | Duration$ UntilTheEndOfYourNextTurn | ForgetOnMoved$ Exile | SubAbility$ DBCleanup | SpellDescription$ You may play up to two of those cards until the end of your next turn.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/ghitu_embercoiler.txt:9]: Fix description / usage

File: forge-gui/res/cardsfolder/upcoming/ghitu_embercoiler.txt [hunk 2]
```diff
@@ -0,0 +1,13 @@
+Name:Ghitu Embercoiler
+ManaCost:1 R
+Types:Creature Human Wizard
+PT:2/2
+K:Prowess
+T:Mode$ Phase | Phase$ Main1 | PreCombatMain$ True | ValidPlayer$ You | TriggerZones$ Battlefield | Execute$ TrigSeek | TriggerDescription$ At the beginning of your precombat main phase, you may discard a card. If you do, seek a card with greater mana value and exile it. Until the end of your next turn, you may play the exiled card. 
+SVar:TrigSeek:AB$ ChangeZone | Cost$ Discard<1/Card> | Origin$ Library | Destination$ Exile | AtRandom$ True | NoShuffle$ True | Mandatory$ True | NoLooking$ True | NoReveal$ True | ChangeType$ Card.cmcGTX | ChangeNum$ 1 | RememberChanged$ True | SubAbility$ DBEffect
+SVar:X:Discarded$CardManaCost
+SVar:DBEffect:DB$ Effect | RememberObjects$ Remembered | StaticAbilities$ MayPlay | Duration$ UntilTheEndOfYourNextTurn | ForgetOnMoved$ Exile | SubAbility$ DBCleanup | SpellDescription$ Until the end of your next turn, you may play the exiled card.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/ghitu_embercoiler.txt:9]: You really don't need SpellDescription on a trigger ;)

File: forge-gui/res/cardsfolder/upcoming/slimefoot_thallid_transplant .txt [hunk 3]
```diff
@@ -0,0 +1,11 @@
+Name:Slimefoot, Thallid Transplant
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/slimefoot_thallid_transplant .txt:1]: Fix filename

File: forge-gui/res/cardsfolder/upcoming/coalition_construct.txt [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+SVar:TrigEffect:DB$ Effect | RememberObjects$ Valid Creature.YouCtrl+Other+ChosenType,ValidHand Creature.YouOwn+Other+ChosenType | StaticAbilities$ PerpetualP1P1 | Name$ Coalition Constructs's Perpetual Effect | Duration$ Permanent
+SVar:PerpetualP1P1:Mode$ Continuous | Affected$ Card.IsRemembered | AddPower$ 1 | AddToughness$ 1 | EffectZone$ Command | AffectedZone$ Battlefield,Hand,Graveyard,Exile,Stack,Library,Command | Description$ Other creatures you control of the chosen type and creature cards of that type in your hand perpetually get +1/+1.
+SVar:Update:Mode$ ChangesZone | Origin$ Any | Destination$ Any | Static$ True | ValidCard$ Card.IsRemembered | Execute$ DBUpdate
+SVar:DBUpdate:DB$ UpdateRemember
+Oracle:As Coalition Construct enters the battlefield choose a creature type.\nCoalition Construct is the chosen type in addition to its other types\nWhen Coalition Construct enters the battlefield, other creatures you control of the chosen type and creature cards of that type in your hand perpetually get +1/+1.
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/coalition_construct.txt:13]: `types\` add .

File: forge-gui/res/cardsfolder/upcoming/ghitu_embercoiler.txt [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+Types:Creature Human Wizard
+PT:2/2
+K:Prowess
+T:Mode$ Phase | Phase$ Main1 | PreCombatMain$ True | ValidPlayer$ You | TriggerZones$ Battlefield | Execute$ TrigSeek | TriggerDescription$ At the beginning of your precombat main phase, you may discard a card. If you do, seek a card with greater mana value and exile it. Until the end of your next turn, you may play the exiled card. 
+SVar:TrigSeek:AB$ ChangeZone | Cost$ Discard<1/Card> | Origin$ Library | Destination$ Exile | AtRandom$ True | NoShuffle$ True | Mandatory$ True | NoLooking$ True | NoReveal$ True | ChangeType$ Card.cmcGTX | ChangeNum$ 1 | RememberChanged$ True | SubAbility$ DBEffect
+SVar:X:Discarded$CardManaCost
+SVar:DBEffect:DB$ Effect | RememberObjects$ Remembered | StaticAbilities$ MayPlay | Duration$ UntilTheEndOfYourNextTurn | ForgetOnMoved$ Exile | SubAbility$ DBCleanup
+SVar:MayPlay:Mode$ Continuous | Affected$ Card.IsRemembered | MayPlay$ True | EffectZone$ Command | AffectedZone$ Exile | Description$ Until the end of your next turn, you may play the exiled card. 
+SVar:DBCleanup:DB$ Cleanup | ClearRemembered$ True
+DeckHas:Ability$Graveyard|Discard
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/ghitu_embercoiler.txt:12]: why Graveyard?

File: forge-gui/res/cardsfolder/upcoming/juggernaut_peddler.txt [hunk 6]
```diff
@@ -0,0 +1,12 @@
+Name:Juggernaut Peddler
+ManaCost:W B
+Types:Creature Human Artificer 
+PT:2/2
+K:Vigilance
+T:Mode$ ChangesZone | Affected$ Card.Self | Origin$ Any | Destination$ Battlefield | Execute$ TrigDiscard | SpellDescription$ When Juggernaut Peddler enter the battlefield, target player reveals all nonland cards in their hand. You may choose one of those cards. If you do, that player exiles it and conjures a card name Juggernaut into their hand.
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/juggernaut_peddler.txt:6]: `card name` > `card named`

File: forge-gui/res/cardsfolder/upcoming/juggernaut_peddler.txt [hunk 7]
```diff
@@ -0,0 +1,12 @@
+Name:Juggernaut Peddler
+ManaCost:W B
+Types:Creature Human Artificer 
+PT:2/2
+K:Vigilance
+T:Mode$ ChangesZone | Affected$ Card.Self | Origin$ Any | Destination$ Battlefield | Execute$ TrigDiscard | SpellDescription$ When Juggernaut Peddler enter the battlefield, target player reveals all nonland cards in their hand. You may choose one of those cards. If you do, that player exiles it and conjures a card name Juggernaut into their hand. 
+SVar:TrigDiscard:DB$ Reveal | ValidTgts$ Player | TgtPrompt$ Select target player | RevealAllValid$ Card.nonLand+TargetedPlayerCtrl | SubAbility$ DBExile | StackDescription$ SpellDescription | SpellDescription$ Target player reveals all cards with mana value 2 or greater in their hand. You choose a card from among those cards. Exile that card. If a card was exiled this way, that player drafts a card from CARDNAME's spellbook and reveals it.
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/juggernaut_peddler.txt:7]: Remove this Stack and SpellDesc

Comment (Northmoc) [reply forge-gui/res/cardsfolder/upcoming/juggernaut_peddler.txt:7]: Also TgtPrompt not needed

File: forge-gui/res/cardsfolder/upcoming/nantuko_slicer.txt [hunk 8]
```diff
@@ -0,0 +1,14 @@
...
+SVar:PerpetualAbility:Mode$ Continuous | AddStaticAbility$ SpendAnyMana | Affected$ Card.IsRemembered | EffectZone$ Command | AffectedZone$ Battlefield,Hand,Graveyard,Exile,Stack,Library,Command | Description$ The conjured card perpetually gains "You may spend mana as though it were mana of any color to cast this spell."
+SVar:SpendAnyMana:Mode$ Continuous | Affected$ Card.Self | EffectZone$ All | AffectedZone$ Stack | AddHiddenKeyword$ May spend mana as though it were mana of any color to cast CARDNAME | Description$ You may spend mana as though it were mana of any color to cast this spell.
+SVar:DBCleanup:DB$ Cleanup | ClearRemembered$ True
+DeckHas:Ability$Graveyard
+Oracle:Kicker{B}\nWhen Nantuko Slicer enters the battlefield return target card from your graveyard to you hand. If this spell was kicked, conjure a dubplicate of target card in an opponent's graveyard into your hand. It perpetually gains:"You may spend mana as though it were mana of any color to cast this spell."
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/nantuko_slicer.txt:14]: `gains:"You` `gains "You`
dubplicate (spelling)

File: forge-gui/res/cardsfolder/upcoming/nantuko_slicer.txt [hunk 9]
```diff
@@ -0,0 +1,14 @@
+Name:Nantuko Slicer
+ManaCost:2 G G
+Types:Creature Insect
+PT:3/2
+K:Kicker:B
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigChangeZone | OptionalDecider$ You | TriggerDescription$ When CARDNAME enters the battlefield, return target card from your graveyard to you hand. If this spell was kicked, conjure a dubplicate of target card in an opponent's graveyard into your hand. It perpetually gains:"You may spend mana as though it were mana of any color to cast this spell."
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/nantuko_slicer.txt:6]: `gains:"You` `gains "You`
dubplicate (spelling)

File: forge-gui/res/cardsfolder/upcoming/nantuko_slicer.txt [hunk 10]
```diff
@@ -0,0 +1,14 @@
...
+PT:3/2
+K:Kicker:B
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigChangeZone | OptionalDecider$ You | TriggerDescription$ When CARDNAME enters the battlefield, return target card from your graveyard to you hand. If this spell was kicked, conjure a dubplicate of target card in an opponent's graveyard into your hand. It perpetually gains:"You may spend mana as though it were mana of any color to cast this spell." 
+SVar:TrigChangeZone:DB$ ChangeZone | Origin$ Graveyard | Destination$ Hand | ValidTgts$ Card.YouOwn | SubAbility$ DBConjure
+SVar:DBConjure:DB$ MakeCard | Condition$ Kicked | Conjure$ True | TgtPrompt$ Select target creature card in an opponent's graveyard | ValidTgts$ Creature.OppOwn+inZoneGraveyard | TgtZone$ Graveyard | DefinedName$ ThisTargetedCard | Zone$ Hand | RememberMade$ True | SubAbility$ DBEffect 
+SVar:DBEffect:DB$ Effect | RememberObjects$ Remembered | StaticAbilities$ PerpetualAbility | Duration$ Permanent | Name$ Nantuko Slicer's Perpetual Effect | SubAbility$ DBCleanup | SpellDescription$ It perpetually gains "You may spend mana as though it were mana of any color to cast this spell."
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/nantuko_slicer.txt:9]: SpellDescription for sub of trigger is not used

File: forge-gui/res/cardsfolder/upcoming/nantuko_slicer.txt (hunk 9 above)
Comment (Northmoc) [reply forge-gui/res/cardsfolder/upcoming/nantuko_slicer.txt:6]: you hand > your hand

File: forge-gui/res/cardsfolder/upcoming/nantuko_slicer.txt (hunk 8 above)
Comment (Northmoc) [reply forge-gui/res/cardsfolder/upcoming/nantuko_slicer.txt:14]: you hand > your hand

File: forge-gui/res/cardsfolder/upcoming/nantuko_slicer.txt (hunk 10 above)
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/nantuko_slicer.txt:9]: this should also have Condition

File: forge-gui/res/cardsfolder/upcoming/tiana_angel_mechanic.txt [hunk 11]
```diff
@@ -0,0 +1,10 @@
+Name:Tiana, Angel Mechanic 
+ManaCost:1 R W
+Types:Legendary Creature Angel Artificer
+PT:3/2
+K:Flying
+T:Mode$ Crewed | ValidCrew$ Card.Self,Creature.Legendary+Other+YouCtrl | Execute$ TrigEffect | TriggerZones$ Battlefield | TriggerDescription$ Whenever CARDNAME or another legendary creature you control crews a vehicle, that vehicle perpetually gains +1/+0.
+SVar:TrigEffect:DB$ Effect | RememberObjects$ TriggeredVehicle | StaticAbilities$ PerpetualP1P1 | Name$ Tiana, Angel Mechanic's Perpetual Effect | Duration$ Permanent
+SVar:PerpetualP1P1:Mode$ Continuous | Affected$ Card.IsRemembered | AddPower$ 1 | EffectZone$ Command | AffectedZone$ Battlefield,Hand,Graveyard,Exile,Stack,Library,Command | Description$ This vehicle perpetually gains +1/+0.
+DeckHints:Type$Vehicle|Legendary
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/tiana_angel_mechanic.txt:9]: probably Needs Vehicle

File: forge-gui/res/cardsfolder/upcoming/reezug_the_bonecobbler.txt [hunk 12]
```diff
@@ -0,0 +1,11 @@
...
+PT:1/3
+A:AB$ Effect | Name$ Reezug, the Bonecobbler's Effect | Cost$ T | ValidTgts$ Creature.YouOwn | TgtZone$ Graveyard | RememberObjects$ Targeted | TgtPrompt$ Select target creature card in your graveyard | StaticAbilities$ MayPlay| SubAbility$ DBEffect | SpellDescription$ Target creature in your graveyard perpetually becomes an artifact. You may cast that card this turn. 
+SVar:DBEffect:DB$ Effect | Name$ Reezug, the Bonecobbler's Effect | RememberObjects$ Targeted | StaticAbilities$ PerpetualArtifact | Duration$ Permanent 
+SVar:PerpetualArtifact:Mode$ Continuous | Affected$ Card.IsRemembered | AddType$ Artifact | RemoveCardTypes$ True | Description$ This creature perpetually becomes an artifact. (It loses all other card types.)
+SVar:MayPlay:Mode$ Continuous | MayPlay$ True | EffectZone$ Command | Affected$ Card.IsRemembered | AffectedZone$ Graveyard | Description$ Until end of turn, you may cast that card. 
+DeckHints:Ability$Graveyard & Type$Artifact 
+DeckHas:Ability$Graveyard & Type$Artifact 
+Oracle:{T}:Target creature in your graveyard perpetually becomes an artifact. You may cast that card this turn. (It loses all other card types.)
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/reezug_the_bonecobbler.txt:11]: Is it not "creature **card** in your graveyard" ?

File: forge-gui/res/cardsfolder/upcoming/reezug_the_bonecobbler.txt [hunk 13]
```diff
@@ -0,0 +1,11 @@
+Name:Reezug, the Bonecobbler
+ManaCost:1 B
+Types:Legendary Creature Spirit Artificer
+PT:1/3
+A:AB$ Effect | Name$ Reezug, the Bonecobbler's Effect | Cost$ T | ValidTgts$ Creature.YouOwn | TgtZone$ Graveyard | RememberObjects$ Targeted | TgtPrompt$ Select target creature card in your graveyard | StaticAbilities$ MayPlay| SubAbility$ DBEffect | SpellDescription$ Target creature in your graveyard perpetually becomes an artifact. You may cast that card this turn. 
+SVar:DBEffect:DB$ Effect | Name$ Reezug, the Bonecobbler's Effect | RememberObjects$ Targeted | StaticAbilities$ PerpetualArtifact | Duration$ Permanent
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/reezug_the_bonecobbler.txt:6]: Perpetual Effect

File: forge-gui/res/cardsfolder/upcoming/reezug_the_bonecobbler.txt [hunk 14]
```diff
@@ -0,0 +1,11 @@
+Name:Reezug, the Bonecobbler
+ManaCost:1 B
+Types:Legendary Creature Spirit Artificer
+PT:1/3
+A:AB$ Effect | Name$ Reezug, the Bonecobbler's Effect | Cost$ T | ValidTgts$ Creature.YouOwn | TgtZone$ Graveyard | RememberObjects$ Targeted | TgtPrompt$ Select target creature card in your graveyard | StaticAbilities$ MayPlay| SubAbility$ DBEffect | SpellDescription$ Target creature in your graveyard perpetually becomes an artifact. You may cast that card this turn.
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/reezug_the_bonecobbler.txt:5]: MayPlay| spacing

File: forge-gui/res/cardsfolder/upcoming/reezug_the_bonecobbler.txt [hunk 15]
```diff
@@ -0,0 +1,11 @@
+Name:Reezug, the Bonecobbler
+ManaCost:1 B
+Types:Legendary Creature Spirit Artificer
+PT:1/3
+A:AB$ Effect | Name$ Reezug, the Bonecobbler's Effect | Cost$ T | ValidTgts$ Creature.YouOwn | TgtZone$ Graveyard | RememberObjects$ Targeted | TgtPrompt$ Select target creature card in your graveyard | StaticAbilities$ MayPlay| SubAbility$ DBEffect | SpellDescription$ Target creature in your graveyard perpetually becomes an artifact. You may cast that card this turn. 
+SVar:DBEffect:DB$ Effect | Name$ Reezug, the Bonecobbler's Effect | RememberObjects$ Targeted | StaticAbilities$ PerpetualArtifact | Duration$ Permanent 
+SVar:PerpetualArtifact:Mode$ Continuous | Affected$ Card.IsRemembered | AddType$ Artifact | RemoveCardTypes$ True | Description$ This creature perpetually becomes an artifact. (It loses all other card types.)
+SVar:MayPlay:Mode$ Continuous | MayPlay$ True | EffectZone$ Command | Affected$ Card.IsRemembered | AffectedZone$ Graveyard | Description$ Until end of turn, you may cast that card.
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/reezug_the_bonecobbler.txt:8]: Description seems weird

File: forge-gui/res/cardsfolder/upcoming/reezug_the_bonecobbler.txt (hunk 14 above)
Comment (Northmoc) [reply forge-gui/res/cardsfolder/upcoming/reezug_the_bonecobbler.txt:5]: Name will be generated automatically

File: forge-gui/res/cardsfolder/upcoming/coalition_construct.txt [hunk 16]
```diff
@@ -0,0 +1,13 @@
+Name:Coalition Construct
+ManaCost:3
+Types:Artifact Creature Construct
+PT:2/2
+K:ETBReplacement:Other:ChooseCT
+SVar:ChooseCT:DB$ ChooseType | Type$ Creature | AILogic$ MostProminentInComputerDeck | SpellDescription$ As CARDNAME enters the battlefield, choose a creature type.
+S:Mode$ Continuous | EffectZone$ All | Affected$ Card.Self | CharacteristicDefining$ True | AddType$ ChosenType | Description$ CARDNAME is the chosen type in addition to its other types.
+T:Mode$ ChangesZone | ValidCard$ Card.Self | Origin$ Any | Destination$ Battlefield | Execute$ TrigEffect| TriggerDescription$ When CARDNAME enters the battlefield, other creatures you control of the chosen type and creature cards of that type in your hand perpetually get +1/+1.
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/coalition_construct.txt:8]: TrigEffect| spacing

File: forge-gui/res/cardsfolder/upcoming/reezug_the_bonecobbler.txt [hunk 17]
```diff
@@ -0,0 +1,11 @@
...
+PT:1/3
+A:AB$ Effect | Name$ Reezug, the Bonecobbler's Perpetual Effect | Cost$ T | ValidTgts$ Creature.YouOwn | TgtZone$ Graveyard | RememberObjects$ Targeted | TgtPrompt$ Select target creature card in your graveyard | StaticAbilities$ MayPlay | SubAbility$ DBEffect | SpellDescription$ Target creature card in your graveyard perpetually becomes an artifact. You may cast that card this turn. 
+SVar:DBEffect:DB$ Effect | Name$ Reezug, the Bonecobbler's Perpetual Effect | RememberObjects$ Targeted | StaticAbilities$ PerpetualArtifact | Duration$ Permanent 
+SVar:PerpetualArtifact:Mode$ Continuous | Affected$ Card.IsRemembered | AddType$ Artifact | RemoveCardTypes$ True | Description$ This creature perpetually becomes an artifact. (It loses all other card types.)
+SVar:MayPlay:Mode$ Continuous | MayPlay$ True | EffectZone$ Command | Affected$ Card.IsRemembered | AffectedZone$ Graveyard | Description$ You may cast that card this turn.
+DeckHints:Ability$Graveyard & Type$Artifact 
+DeckHas:Ability$Graveyard & Type$Artifact 
+Oracle:{T}:Target creature card in your graveyard perpetually becomes an artifact. You may cast that card this turn. (It loses all other card types.)
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/reezug_the_bonecobbler.txt:11]: space after `{T}:`

File: forge-gui/res/cardsfolder/upcoming/reezug_the_bonecobbler.txt [hunk 18]
```diff
@@ -0,0 +1,11 @@
+Name:Reezug, the Bonecobbler
+ManaCost:1 B
+Types:Legendary Creature Spirit Artificer
+PT:1/3
+A:AB$ Effect | Name$ Reezug, the Bonecobbler's Perpetual Effect | Cost$ T | ValidTgts$ Creature.YouOwn | TgtZone$ Graveyard | RememberObjects$ Targeted | TgtPrompt$ Select target creature card in your graveyard | StaticAbilities$ MayPlay | SubAbility$ DBEffect | SpellDescription$ Target creature card in your graveyard perpetually becomes an artifact. You may cast that card this turn.
```
[the remark this answers is not in the document]
Comment (Northmoc) [reply forge-gui/res/cardsfolder/upcoming/reezug_the_bonecobbler.txt:5]: Uh line 5 should not be perpetual... just get rid of the Name param, it's automatic

[the remark this answers is not in the document]
Comment (Northmoc) [reply forge-gui/res/cardsfolder/upcoming/reezug_the_bonecobbler.txt:5]: Weird that you are executing the effects in the opposite order of the oracle

File: forge-gui/res/cardsfolder/upcoming/nantuko_slicer.txt [hunk 19]
```diff
@@ -0,0 +1,14 @@
...
+SVar:DBEffect:DB$ Effect | RememberObjects$ Remembered | Condition$ Kicked | StaticAbilities$ PerpetualAbility | Duration$ Permanent | Name$ Nantuko Slicer's Perpetual Effect | SubAbility$ DBCleanup
+SVar:PerpetualAbility:Mode$ Continuous | AddStaticAbility$ SpendAnyMana | Affected$ Card.IsRemembered | EffectZone$ Command | AffectedZone$ Battlefield,Hand,Graveyard,Exile,Stack,Library,Command | Description$ The conjured card perpetually gains "You may spend mana as though it were mana of any color to cast this spell."
+SVar:SpendAnyMana:Mode$ Continuous | Affected$ Card.Self | EffectZone$ All | AffectedZone$ Stack | AddHiddenKeyword$ May spend mana as though it were mana of any color to cast CARDNAME | Description$ You may spend mana as though it were mana of any color to cast this spell.
+SVar:DBCleanup:DB$ Cleanup | ClearRemembered$ True
+DeckHas:Ability$Graveyard
+Oracle:Kicker{B}\nWhen Nantuko Slicer enters the battlefield return target card from your graveyard to your hand. If this spell was kicked, conjure a duplicate of target card in an opponent's graveyard into your hand. It perpetually gains: "You may spend mana as though it were mana of any color to cast this spell."
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/nantuko_slicer.txt:14]: Is there really a `:` after gains? That's unlike any similar card

File: forge-gui/res/cardsfolder/upcoming/juggernaut_peddler.txt [hunk 20]
```diff
@@ -0,0 +1,12 @@
+Name:Juggernaut Peddler
+ManaCost:W B
+Types:Creature Human Artificer 
+PT:2/2
+K:Vigilance
+T:Mode$ ChangesZone | Affected$ Card.Self | Origin$ Any | Destination$ Battlefield | Execute$ TrigDiscard | SpellDescription$ When Juggernaut Peddler enters the battlefield, target player reveals all nonland cards in their hand. You may choose one of those cards. If you do, that player exiles it and conjures a card named Juggernaut into their hand.
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/juggernaut_peddler.txt:6]: When CARDNAME

**Comment by Simisays** [comment]
@Northmoc Thanks for all your comments. Shall try to make my next cards cleaner to save you some work

**Review by tool4ever — APPROVED** [review]

**Pull request merged**